### DISCUSSION TITLE
Mobile breakpoints do not work in Gatsby

### DISCUSSION THREAD

**Discussion by ThePaulo1** [tailwindlabs/tailwindcss#7338] [Help] [answered]
I am building a custom navbar in Gatsby using TailwindCss for styling. For reference I am following a tutorial on YouTube which's code can be found [here][1].

When I use the mobile breakpoints as prefix like in the example below only the div saying "mobile" works as it should be:

```js
<nav>
    <div className={"flex"}>
        <div className={"hidden md:flex"}>desktop</div>
        <div className={"md:hidden"}>mobile</div>
    </div>
</nav>
```

The output "desktop" should be shown when the text "mobile" vanishes but for some reason it doesn't. Am I doing something wrong or is there a problem with Tailwind?

[![enter image description here][2]][2]

  [1]: https://codepen.io/chris__sev/pen/QWGjQKe?editors=1000
  [2]: https://i.stack.imgur.com/2CoWw.gif

**Comment by adamwathan** [comment]
Hey! Working as expected as far as I can tell:

https://play.tailwindcss.com/PvaLLcuFjF?size=714x720

Can you provide a full reproduction repository? Must be more to the story.

**Reply by ThePaulo1** [reply]
Here is a simplified Gatsby project on [Codesandbox](https://codesandbox.io/s/gatsby-tailwind-pr3kd). The "md:flex" is ignored as in my current project...

**Comment by ThePaulo1** [comment, marked as answer]
Solved the problem by redoing the [installation](https://tailwindcss.com/docs/guides/gatsby) because I've noticed that the autoprefixer plugin was missing in the `postcss.config.js` file. Luckily I came across this while watching one of your absolutely awesome tutorials on YouTube...
